File: threedollar-api/src/main/java/com/depromeet/threedollar/api/service/jwt/dto/component/JwtTokenProviderComponent.java
package com.depromeet.threedollar.api.service.jwt.dto.component;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Setter
@Getter
@ConfigurationProperties(prefix = "jwt")
@NoArgsConstructor
@Component
public class JwtTokenProviderComponent {

	private String secretKey;

	private String issuer;

}

File: threedollar-api/src/main/java/com/depromeet/threedollar/api/service/token/dto/component/JwtTokenProviderComponent.kt
package com.depromeet.threedollar.api.service.token.dto.component

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@ConfigurationProperties(prefix = "jwt")
@Configuration
class JwtTokenProviderComponent {

    lateinit var secretKey: String

    lateinit var issuer: String

}
